po/es.po

```diff
@@ -0,0 +1,1212 @@
+"_Qualification Kit_. You can then provide these to _your_ assessor when "
+"certifying _your_ development process."
+msgstr ""
+"Puedes comprarnos los documentos firmados digitalmente como parte de un "
+"_Kit de Calificación_. Después puedes proveer esos documentos a _tu_ asesor "
+"cuando certifiques _tu_ proceso de desarrollo."
+
+#: src\iso26262.md:74
+msgid "But it's also open source"
+msgstr "Pero también es open source"
+
+#: src\iso26262.md:76
+msgid ""
+"Anyone can inspect the unsigned documents and test matrices in our "
+"open-source repository without a contract."
+msgstr ""
+"Cualquiera puede inspeccionar los documentos no firmados y nuestras pruebas de matrices "
+"en nuestro repositorio open-source sin un contrato"
+
+#: src\iso26262.md:78
+msgid "<https://github.com/ferrocene/ferrocene>"
+msgstr ""
+
+#: src\iso26262.md:80
+msgid "<https://public-docs.ferrocene.dev>"
+msgstr ""
+
+#: src\support.md:3
+msgid "Ferrocene is open source"
+msgstr "Ferrocene es open source"
+
+#: src\support.md:5
+msgid "The Source Code is MIT/Apache-2.0"
+msgstr "El código fuente es MIT/Apache-2.0"
+
+#: src\support.md:6
+msgid ""
+"The [Ferrocene Language Specification](https://spec.ferrocene.dev) is "
+"MIT/Apache-2.0"
+msgstr ""
+"La [Especificación del Lenguaje de Ferrocene](https://spec.ferrocene.dev) es "
+"MIT/Apache-2.0"
+
+#: src\support.md:7
+msgid ""
+"You can even [read the Qualification "
+"Documents](https://public-docs.ferrocene.dev/main/index.html) right now"
+msgstr ""
+"Hasta puedes [leer los Documentos de "
+"Calificación](https://public-docs.ferrocene.dev/main/index.html) ahorita"
+
+#: src\support.md:12
+msgid "And you sell?"
+msgstr "¿Y qué vendes"
+
+#: src\support.md:14
+msgid "Binaries"
+msgstr "Binarios"
+
+#: src\support.md:15
+msgid "Long-term Support"
+msgstr "Soporte de largo plazo"
+
+#: src\support.md:16
+msgid "for Ferrocene"
+msgstr "para Ferrocene"
+
+#: src\support.md:17
+msgid "for third-party crates (as an add-on)"
+msgstr "y para crates de terceros (por aparte)"
+
+#: src\support.md:18
+msgid "Includes digitally-signed Qualification Documents for your assessor"
+msgstr "Incluye los Documentos de Calificación firmados digitalmente para tu asesor"
+
+#: src\support.md:20
+msgid "How long is long-term?"
+msgstr "¿Qué tan largo es largo plazo?"
+
+#: src\support.md:22
+msgid "Upstream Rust only supports each release for six weeks"
+msgstr "Upstream Rust solo da soporte a cada release por 6 semanas"
+
+#: src\support.md:23
+msgid "There are no backports to old stable releases"
+msgstr "No hay backports a releases viejos y estables"
+
+#: src\support.md:24
+msgid "We pick a _rolling_ release, _stabilise_ it, and support it for 2 years"
+msgstr "Escogemos un _rolling_ release, lo _estabilizamos_ y le damos soporte por 2 años"
+
+#: src\support.md:25
+msgid "We give subscribers bug-fix releases as we fix bugs"
+msgstr "Le damos a los suscriptores releases de bug-fixes mientras los encontramos"
```

Comment:
I'd translate this as "Le damos a los suscriptores releases de bug-fixes a medida que los encontramos"

Reply:
Excelente,lo prefiero.